LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/JsonHandlers/LnsStationConfiguration.cs

```diff
@@ -160,6 +160,11 @@ private class Sx1301Config
         /// </summary>
         private const SpreadingFactor FskSpreadingFactor = 0;
 
+        /// <summary>
+        /// In the router configuration, you can skip a DataRate setting by specifying a spreading factor of -1.
+        /// </summary>
+        private const SpreadingFactor SkipSpreadingFactor = (SpreadingFactor)(-1);
```

Comment:
```suggestion
        private const SpreadingFactor SkipSpreadingFactor = SpreadingFactor.UNDEFINED;
```

Reply:
This is not applicable to latest changes. We decided to not extend SpreadingFactor with UNDEFINED as from previous comments from @atifaziz